### ISSUE TITLE
[flowbroker-ui] closing tab

### ISSUE THREAD

**Issue by betefaber** [dojot/dojot#2397]
* **I'm submitting a ...**
  - [ ] bug report
  - [x] feature request
  - [ ] support request

* **Do you want to request a *feature* or report a *bug*?**

* **What is the current behavior?**
When a flow is added a new tab opens.
There is no way to close a tab other than deleting the flow. Is the flow really removed? 

* **If the current behavior is a bug, please provide the steps to reproduce and if possible a minimal demo of the problem**

![Captura de tela de 2021-12-16 19-15-12](https://user-images.githubusercontent.com/37310063/146457566-29529c33-23d1-4066-a453-5571ab02dd5b.png)

- delete flow

![Captura de tela de 2021-12-16 19-15-39](https://user-images.githubusercontent.com/37310063/146457375-aed1ce6b-6478-42e2-9361-88343719f5b5.png)

* **What is the expected behavior?**

* **What is the motivation / use case for changing the behavior?**

* **Please tell us about your environment:**
  
  - Version: flowbroker-ui/development
  - Environment: docker-compose
  - Operating system: Ubuntu 18.04

* **Other information** (e.g. detailed explanation, stacktraces, related issues, suggestions how to fix, links for us to have context, eg. stackoverflow, gitter, etc)

**Comment by cfrancisco** [comment]
Exactly! The tabs mean the installed flows, so the only way to 'close it' is to remove it.
